### DISCUSSION TITLE
Can’t access sub/sub folder with reverse proxy

### DISCUSSION THREAD

**Discussion by Cetartiodactyla** [bpatrik/pigallery2#366] [General]
Hello,

I set up a reverse proxy (apache) for my gallery. I can’t navigate to a sub/sub folder like https://photos.mydomain.com/gallery/subfolder1%2Fsubfolder2 instead it directs me back to https://photos.mydomain.com/gallery/. No problem while accessing the gallery through the specified port though.

http://photos.mydomain.com:3333/gallery/subfolder1%2Fsubfolder2 ✅  Works
http://photos.mydomain.com/gallery/subfolder1%2Fsubfolder2 ❌  Doesn’t work

Apache directives for subdomain `photos.mydomain.com`:

```
    ProxyPass / http://mydomain.com:3333/
    ProxyPassReverse / http://mydomain.com:3333/
```

This might be a misconfiguration of the reverse proxy. Any advice on this?

Thanks,
Florian

**Comment by bpatrik** [comment]
Unfortunately, I do not have much experience with apache. You can try looking through the issues related to apache: https://github.com/bpatrik/pigallery2/issues?q=apache

Maybe some of them offer some help.
Also the app uses by default nginx. There is also a sample config available for that in this repo.

**Comment by wizgod** [comment]
@floriangeyer I had the same problem and couldn't get it working; instead I resorted to creating a url redirect to the port instead.

**Reply by Cetartiodactyla** [reply]
This is probably the better solution: In Apache Configuration set `AllowEncodedSlashes On`

**Reply by wizgod** [reply]
@floriangeyer I reenabled the proxy and set `AllowEncodedSlashes On` and it worked like a charm, thanks!
